docs/components.md

```diff
@@ -7,9 +7,9 @@
 | Component | Description | Repository |
 | :-------- | :---------- | :--------- |
 | Sugar | Activity Menu, Journal, Network View and Control Panel | https://github.com/sugarlabs/sugar |
-| Toolkit | Activity, Journal, and Plaform library | https://github.com/sugarlabs/sugar-toolkit-gtk3 |
-| Datastore | Journal Storage API | https://github.com/sugarlabs/sugar-datastore |
-| Artwork | Icons, Themes and Cursors | https://github.com/sugarlabs/sugar-artwork |
+| Sugar Toolkit | Activity, Journal, and Plaform library | https://github.com/sugarlabs/sugar-toolkit-gtk3 |
+| Sugar Datastore | Journal Storage API | https://github.com/sugarlabs/sugar-datastore |
```

Comment:
I think this should be `Sugar Toolkit Gtk3` seeing as we have an existing `sugar-toolkit`.

Reply:
https://github.com/sugarlabs/sugar/pull/954/commits/7fbc94e438208dc23ed27f6a9c33d5fa5204ec94 adds this change, but I'm not sure it helps, and it ties us to a release version range.